pom.xml

```diff
@@ -743,7 +743,7 @@
             <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-lang3</artifactId>
-                <version>3.14.0</version>
+                <version>3.17.0</version>
```

Comment:
Need to look into this.

Reply:
[`US969006`](https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=969006) raised for this.